### DISCUSSION TITLE
Error when trying to CalcFields a non-FlowField field

### DISCUSSION THREAD

**Discussion by invalidCards** [StefanMaron/BusinessCentral.LinterCop#177] [New Rule]
Using `CalcFields()` on a field that is not a FlowField is almost always a programmer error and should ideally be caught at compile time to prevent runtime errors from occurring, however this error is not present within Microsoft's own rules. Would it be possible to add an error when attempting to use a table field that is not a FlowField within a `CalcFields()` call?

**Comment by ChrisBlankDe** [comment]
CodeCop Warning AA0211 catches this:
![image](https://user-images.githubusercontent.com/18051656/203637384-b628161b-031a-4089-9001-0f792d8490c9.png)

https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/analyzers/codecop-aa0211
